pkg/config/wikidata.go

```diff
@@ -55,7 +55,11 @@ var wikidata = struct {
 	// nopronom determines whether the identifier will be build without
 	// patterns from PRONOM sources outside of Wikidata.
 	nopronom bool
-	// sourceField provides a way to represent provenance as a new field
+	// provfield lists the field for which we want to retrieve
+	// provenance for in the Wikidata file format SPARQL result. This
+	// should be the subject/IRI to the file format record in Wikidata.
+	provfield string
```

Comment:
Need to iron out the naming conventions here. As noted above, `provfield` is actually a SPARQL ?variable and it is also translated to key in `wikiprov`. Key works for me so that it isn't confused with variables in general. Of course, `sparqlVariable` may be cleaner and more accurate. Perhaps that.